contracts/sol6/KyberStorage.sol

```diff
@@ -191,12 +191,12 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
 
         // update reserve mappings
         reserveIdToAddresses[reserveId].push(reserveIdToAddresses[reserveId][0]);
-        reserveIdToAddresses[reserveId][0] = address(0);
-        reserveAddressToId[reserve] = bytes32(0);
+        delete reserveIdToAddresses[reserveId][0];
```

Comment:
It seems that the logic in here is not tested yet

Reply:
actually this part is wrong
will reverse this change.